inference/mlflow-huggingface/src/models/bert-base-uncased/model.yaml

```diff
@@ -0,0 +1,14 @@
+name: bert-base-uncased                                               #Required
+path:                                                                 #Required
+  # local_path: "./local-path-to-model-files"                         #Required  Either package or local_path should be passed. if both are given, local_path is given preference  
+  package:                                                            #Required. Either package or local_path should be passed. if both are given, local_path is given preference
+    commit_hash: 5546055f03398095e385d7dc625e636cc8910bf2             #Optional
+    url: https://huggingface.co/bert-base-uncased                     #Required
```

Comment:
> url: https://huggingface.co/bert-base-uncased [](http://example.com/codeflow?start=3&length=46)

We need to add metadata here that captures that we need to do `git lfs` and not a http/ftp based download.

@novaturient95 Could you please add in your PR?

Reply:
We could add that metadata to model.yaml. In that case we can conditionally install LFS. I'll inform my team regarding this and get back to you. All the huggingface stock models require LFS support, hence we went with this generalization.

Reply:
Hello Sasidhar. Huggingface document on downloading models says : https://huggingface.co/docs/hub/models-downloading 
```
git lfs install
git clone <MODEL URL>
```
Hence, to simplify this, we can install LFS for all models.